### ISSUE TITLE
"Packages" has a "Packages" in the tree

### ISSUE THREAD

**Issue by Codeberg-AsGithubAlternative-buhtz** [doxygen/doxygen#9192]
In the tree of my generated documentation the node/entry `Packages` appears two times.

![image](https://user-images.githubusercontent.com/11861496/157206006-33bc39d9-7d50-4ddd-a340-a0414febfd20.png)

Sidenode: This is not about `mypackage` which I know why it appears two times in the tree.

The full project including generated html is in that archive: [project.zip](https://github.com/doxygen/doxygen/files/8204516/project.zip).
.

I use doxygen `1.9.4 (57efad44bfc02744ba5fd8ea820b54a2443e3771*)` from git on Debian 11 (`Linux TONNE 5.10.0-11-amd64 #1 SMP Debian 5.10.92-1 (2022-01-18) x86_64 GNU/Linux`) with Python 3.9.

This is the file and folder structure of the project:
```
project
├── docs
│   ├── Doxyfile
│   └── html
│       ├── ...
│       └── ...
├── LICENSE
├── README.md
├── src
│   ├── mypackage
│   │   ├── __init__.py
│   │   └── mypackage.py
│   ├── setup.cfg
│   └── setup.py
└── tests
```

This is `doxygen -x`:
```
# Difference with default Doxyfile 1.9.4 (57efad44bfc02744ba5fd8ea820b54a2443e3771*)
PROJECT_NAME           = Project
OPTIMIZE_OUTPUT_JAVA   = YES
EXTRACT_ALL            = YES
EXTRACT_PRIVATE        = YES
EXTRACT_STATIC         = YES
EXTRACT_LOCAL_METHODS  = YES
SORT_MEMBER_DOCS       = NO
WARN_AS_ERROR          = YES
INPUT                  = ../src/mypackage
FILE_PATTERNS          =
RECURSIVE              = YES
EXCLUDE                = ./_trash
EXAMPLE_PATTERNS       =
HTML_TIMESTAMP         = YES
GENERATE_TREEVIEW      = YES
MATHJAX_RELPATH        = https://cdn.jsdelivr.net/npm/mathjax@3
GENERATE_LATEX         = NO
HAVE_DOT               = YES
UML_LOOK               = YES
DOT_UML_DETAILS        = YES
```

**Comment by albert-github** [comment]
Which version did you use, 1.9.4 is the current master so the full id is necessary from `doxygen -v`

**Comment by Codeberg-AsGithubAlternative-buhtz** [comment]
Dear Albert,
thanks for asking back.

> Which version did you use, 1.9.4 is the current master so the full id is necessary from `doxygen -v`

It is `1.9.4 (57efad44bfc02744ba5fd8ea820b54a2443e3771*)`. You can see the ID in the `doxygen -x` output but I also updated the whole Issue.

**Comment by albert-github** [comment]
Oops, I missed the `doxygen -x` output.

**Comment by Codeberg-AsGithubAlternative-buhtz** [comment]
I am wondering if I am the only person who observe this phenomenon?
Maybe I did something wrong?

**Comment by albert-github** [comment]
I've just pushed a proposed patch, pull request #9197

**Comment by Codeberg-AsGithubAlternative-buhtz** [comment]
> I've just pushed a proposed patch, pull request #9197

Not gone in my case. https://github.com/doxygen/doxygen/pull/9197#issuecomment-1064546676

**Comment by albert-github** [comment]
For completeness: Discussion is running at with the pull request #9197, looks like patch has not been applied locally.

**Comment by albert-github** [comment]
Code has been integrated in master on GitHub (please don't close the issue as this will be done at the moment of an official release).

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.9.4.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
